azext_iot/_help.py

```diff
@@ -352,129 +366,141 @@
 
 helps['iot dps enrollment'] = """
     type: group
-    short-summary: Manage enrollments in an Azure IoT Hub device provisioning service
+    short-summary: Manage enrollments in an Azure IoT Hub Device Provisioning Service.
```

Comment:
@YingXue  Can you validate the changes in character casing is OK for DPS? We have discussed rules for casing in the past for DPS.

Reply:
The file used a mix of casing so I went with the proper noun for the service ... if that makes sense.

Reply:
Sorry, I missed this comment, but the original case is following the guidelines. Anything that starts with an article (an or the in this case) should be lower case.
But seems like the extension if following different rules from Azure CLI Main, I am OK with this change for the consistency.